test/functional/tests/writeback/test_writeback_signals.py

```diff
@@ -0,0 +1,251 @@
+#
+# Copyright(c) 2020 Intel Corporation
+# SPDX-License-Identifier: BSD-3-Clause-Clear
+#
+from time import sleep
+
+import pytest
+
+from api.cas import casadm
+from api.cas.cache_config import CacheMode
+from internal_plugins.scsi_debug import ScsiDebug
+from storage_devices.device import Device
+from storage_devices.disk import DiskType, DiskTypeSet
+from core.test_run import TestRun
+from test_tools.disk_utils import Filesystem
+from test_tools.fs_utils import create_random_test_file
+from test_utils import os_utils
+from test_utils.size import Size, Unit
+
+mount_point = "/mnt/cas"
+log_msg = "/var/log/messages"
+
+
+@pytest.mark.parametrize("cache_mode", [CacheMode.WB, CacheMode.WO])
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+def test_flush_signal_core(cache_mode):
+    """
+        title: Test for FLUSH signals to core device in mode with lazy writes.
+        description: |
+          Test if OpenCAS transmits FLUSH signals to core device in lazy-writes modes.
+        pass_criteria:
+          - FLUSH requests should be passed to core device.
+    """
+    with TestRun.step("Prepare devices for cache and core."):
+        cache_dev = TestRun.disks['cache']
+        cache_dev.create_partitions([Size(256, Unit.MebiByte)])
+        cache_part = cache_dev.partitions[0]
+        scsi_dev = ScsiDebug(
+            {"opts": "1", "dev_size_mb": "768", "sector_size": "512", "physblk_exp": "3"})
+        if os_utils.is_kernel_module_loaded(scsi_dev.module_name):
+            os_utils.unload_kernel_module(scsi_dev.module_name)
+        os_utils.load_kernel_module(scsi_dev.module_name, scsi_dev.params)
```

Comment:
use plugin and then TestRun.scsi_debug_devices will provide you with devices created by the plugin